tests/unit/test_overlay.yaml

```diff
@@ -1,5 +1,7 @@
 description: Charmed Kubernetes overlay for unit testing.
 applications:
+  openstack-integrator:
+    charm: cs:~containers/openstack-integrator
```

Comment:
Curious... Why'd you add `o7k-integrator` to the test when you already had calico in the overlay?

Reply:
i ran into an issue where this charm in the way it's name was formatted was causing something funny to happen i wanted to test for. 

The openstack-integration overlay doesn't specify a charm version for `cs:~containers/openstack-integrator`.  Just that it should be the latest

this opened a bug in shrinkwrap where we were parsing backwards on the last '-'.  In this case -- it was between openstack and integrator.  It didn't show up on other charms like `kubernetes-master-1234` or `calico`